Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event(String description, int id, String from, String to) {
+        super(description, id);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        String status = "[" + getStatusIcon() + "] ";
+        return String.valueOf(this.id) + ". " + "[E]" + status + this.description +
```

Comment:
Nice job breaking into another line for improving code readability here. I think you can consider breaking before an operator just to follow the standard!
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/99804772/7cd96cab-aa03-447a-a700-aa67f5dd9cbe)